pkg/util/util.go

```diff
@@ -133,6 +133,17 @@ func HasKernelBootContainerImage(vmi *v1.VirtualMachineInstance) bool {
 	return true
 }
 
+func HasTPMDevice(vmiSpec *v1.VirtualMachineInstanceSpec) bool {
+	if vmiSpec.Domain.Devices.TPM == nil {
```

Comment:
can you combine these two conditions to a single condition?

Reply:
ok, what about:
```
tpm := vmiSpec.Domain.Devices.TPM
return tpm != nil && (tpm.Enabled == nil || *tpm.Enabled)
```

Reply:
Will do, I guess next we should look at the function right above this :)